patterns/acu_data_compressed.hexpat

```diff
@@ -0,0 +1,49 @@
+#pragma description Assassin's Creed: Unity's Compressed .data file
+#pragma MIME archives/acu_data_compressed
+
+// author = haru233, many thanks to AxCut

```

Comment:
You can use the `#pragma author haru233` pragma here. Then your name will even show up in ImHex